.github/workflows/github-merit-badger.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: github-merit-badger
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  call-action:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: aws-github-ops/github-merit-badger@main
+        id: merit-badger
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          badges: '[beginning-contributor,repeat-contributor,valued-contributor,admired-contributor,star-contributor,distinguished-contributor]'
+          thresholds: '[0,3,6,13,25,50]'
+          badge-type: 'achievement'
```

Comment:
Checking the https://github.com/aws-github-ops/github-merit-badger should the `badge-type` be `AchievementBadger`?

Reply:
As per the doc, it says as `AchievementBadger` is the used class and badge type is `achievement` found this https://github.com/aws-github-ops/github-merit-badger/blob/1bf49a1c4915cfa7b7a58b79806704218b433bbc/.github/workflows/tester.yml#L21.